.github/workflows/deploy-site.yml

```diff
@@ -36,7 +36,7 @@ jobs:
       - name: Setup Node
         uses: actions/setup-node@v3
         with:
-          node-version: "18.10"
+          node-version: "18.16"
```

Comment:
necessary for these 3rd party packages. i believe is a breaking change then to our sdk

Reply:
does everything still work with node 18.16? e2e tests?

Reply:
yeah worked for me